compiler/hie-core/src/Development/IDE/Types/Options.hs

```diff
@@ -14,13 +14,14 @@ module Development.IDE.Types.Options
 
 import Data.Maybe
 import Development.Shake
+import Development.IDE.GHC.Util
 import           GHC hiding (parseModule, typecheckModule)
 import           GhcPlugins                     as GHC hiding (fst3, (<>))
 import qualified Language.Haskell.LSP.Types.Capabilities as LSP
 
 data IdeOptions = IdeOptions
   { optPreprocessor :: GHC.ParsedSource -> ([(GHC.SrcSpan, String)], GHC.ParsedSource)
-  , optGhcSession :: Action HscEnv
+  , optGhcSession :: FilePath -> Action HscEnvEq
   -- ^ Setup a GHC session using a given package state. If a `ParsedModule` is supplied,
```

Comment:
Would be nice to update the comment and mention what the filepath corresponds to.

Reply:
👍